Samples/JavaDependencySampleApp/Sources/JavaDependencySample/main.swift

```diff
@@ -52,4 +58,33 @@ for record in try CSVFormatClass.RFC4180.parse(reader)!.getRecords()! {
   }
 }
 
+print("Now testing Json library...")
+
+let json = Json(#"{"host": "localhost", "port": 80}"#)
```

Comment:
Can we do this in a test in the JavaDependencySample rather than in the main.swift here? In general we'd prefer using more tests and less "everything in main" nowadays